Makefile

```diff
@@ -1,5 +1,13 @@
 # These can be overidden with env vars.
+REGISTRY ?= us.icr.io
+NAMESPACE ?= nyu-devops
```

Comment:
don't we need to change this namespace?

Reply:
You are right, we need. I do it manually before.